onadata/libs/utils/export_tools.py

```diff
@@ -815,6 +844,7 @@ def generate_external_export(  # noqa C901
     filter_query = options.get("query")
     meta = options.get("meta")
     token = options.get("token")
+    sort = options.get("sort")
```

Comment:
@kelvin-muchiri Could we add documentation for this change?

Reply:
@KipSigei Added the [documentation](https://github.com/onaio/onadata/pull/2460/files#diff-194a6eb60a3032894acafae20fb216b5c4c69d6eea231ed18bab3e2186abdd15R393)